### PR TITLE
Mikaela currier

### PR DESCRIPTION
**Pull request by mikaelacurrier**
# Description

Refactored the onboarding page so that the code is more readable. Also added a simple nav bar that points to account, reports, and currently both login and logout. When Auth is completed, this will show either login/logout, never both. Also, logout is not currently hooked up to anything.

Also, on the CreateTeam.js file view, I added the ability to add multiple team members by email on the same component. The component shows a running list of the emails that the user has inputted. Once the user inputs emails, they can also remove incorrect emails on this page. The input is `type=email` but currently any value is accepted. This should not happen.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Change status
- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Test A
tested on local machine
- [ ] Test B

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] There are no merge conflicts

### CODE REVIEW

**Comment by mikaelacurrier** [comment]
I think I'm satisfied with how this looks for now. _Very_ basic styling has been applied and I hope to get this set up to the BE early next week

File: client/src/components/Firebase/Firebase.js [hunk 1]
```diff
@@ -11,20 +13,38 @@ firebase.initializeApp(config);
 
 class Firebase extends React.Component {
 	render() {
-		return <button onClick={this.submit}>GitHub Auth</button>;
+		return (
+			<div>
+				<button onClick={this.submitGitHub}>GitHub Auth</button>
+				{/* <button onClick={this.submitGoogle}>Google Auth</button> */}
```
Comment (erin-koen) [inline client/src/components/Firebase/Firebase.js]: Why's the google part commented out?

File: client/src/components/Onboarding/Onboarding.js [hunk 2]
```diff
@@ -12,70 +17,71 @@ class Onboarding extends Component {
 			emails: []
```
Comment (erin-koen) [inline client/src/components/Onboarding/Onboarding.js]: need joinToggle and createToggle in state? or am i missing something

Comment (erin-koen) [reply client/src/components/Onboarding/Onboarding.js]: weirdly did not show up in GH but i see them in the code

File: client/src/components/Firebase/Firebase.js (hunk 1 above)
Comment (mikaelacurrier) [reply client/src/components/Firebase/Firebase.js]: Doesn't work yet

File: client/src/components/Onboarding/Onboarding.js (hunk 2 above)
Comment (mikaelacurrier) [reply client/src/components/Onboarding/Onboarding.js]: The tab was collapsed, just had to open it up

**Pull request merged**